src/OpenLoco/Town.cpp

```diff
@@ -63,9 +149,22 @@ namespace OpenLoco
      */
     void Town::recalculateSize()
     {
-        registers regs;
-        regs.esi = X86Pointer(this);
-        call(0x004975E0, regs);
+        auto newSize = enumValue(size);
+        if (size < TownSize::metropolis
+            && var_34 >= _populations[enumValue(size) + 1])
```

Comment:
if I understand, `var_34` should be equivalent to the first 2 bytes of `this->pad_34`, so perhaps we could change that padding part from ` uint8_t pad_34[0x38 - 0x34];` into 2 parts `uint16_t var_34;` and `uint8_t pad_36[0x38-0x36];`? I'm unsure what actually this uint16_t represents though

Reply:
nvm I'm wrong

Reply:
this is 100% some kind of population value:
![image](https://user-images.githubusercontent.com/7483209/153710514-151d4cfc-0226-4a0b-9687-8f13443ff7e7.png)
![image](https://user-images.githubusercontent.com/7483209/153710519-72ecd283-87e7-452d-8645-5d153953fd7c.png)

Reply:
I know what `var_34` is - it's the 'next' population count:
![image](https://user-images.githubusercontent.com/7483209/153710999-b84c827d-989f-4bc3-8c35-fc7d6cc47a0f.png)
![image](https://user-images.githubusercontent.com/7483209/153711018-16cc07ec-2956-4fc8-aa01-773e8ee542ca.png)

I basically sat in the debugger waiting for my town to grow, and this `var_34` changes much earlier than the real population, and at some later event, gets 'committed' into the population. The code that does this 'commit' is still un-decompiled, so I have no issues with leaving the name as `var_34` for now